index.js

```diff
@@ -77,6 +77,17 @@ const ContainerService = {
       });
 
       ss.storage.savedConfiguration = object;
+
+      // Maybe rename the Banking container.
+      if (prefService.get("privacy.userContext.enabled") !== true) {
+        let identity = ContextualIdentityService.getIdentityFromId(3);
```

Comment:
How permanent is this renaming code? Is this only needed for Dev/QA because we were all testing beta builds, and new Test Pilot users would never have this issue because they would never have had a "Finance" container, or is this because users may have already been using containers in Nightly?